gapic-generator/lib/gapic/schema/request_param_parser.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "gapic/schema/request_parameter"
+require "gapic/schema/parameter_schema"
+
+module Gapic
+  module Schema
+    # Contains logic for parsing protoc request parameters
+    # from the plugin_opt command line
+    module RequestParamParser
+      class << self
+        # Unescapes a symbol from the string
+        # e.g. "a\.b", '.' => "a.b"
```

Comment:
Probably good to single-quote the string that includes backslashes, just to make it clear to a Rubyist that it's a literal backslash. Same for the comments in the next method.

Also, I'd wrap those strings in backticks so markdown parses them literally and displays them in monospace text rather than possibly trying to parse them.

Reply:
fixed